lib/ibanity/api_schema.ex

```diff
@@ -154,7 +154,7 @@ defmodule Ibanity.ApiSchema do
     after
       result -> result
     else
-      _ -> raise HTTPoison.Error, reason: :timeout
+      error -> IO.inspect(error) && raise HTTPoison.Error, reason: :timeout
```

Comment:
```suggestion
      error -> 
          IO.inspect(error, label: "Fetch API schema")
          raise HTTPoison.Error, reason: :timeout
```

Reply:
what about using Logger instead of IO.inspect()?